spec/spec_helper.rb

```diff
@@ -96,42 +96,42 @@
   #   Kernel.srand config.seed
 end
 
-SimpleCov.start do
-  add_filter %w[
-    acceptance
-    bin
-    config
-    coverage
-    db
-    deploy-eks
-    docker
-    lib
-    log
-    node_modules
-    public
-    spec
-    storage
-    test
-    tmp
-    vendor
-  ]
-  add_group('Components', 'app/components')
-  add_group('Controllers', 'app/controllers')
-  add_group('Dashboards', 'app/dashboard')
-  add_group('Generators', 'app/generators')
-  add_group('Helpers', 'app/helpers')
-  add_group('Jobs', 'app/jobs')
-  add_group('Models', 'app/models')
-  add_group('Presenters', 'app/presenters')
-  add_group('Services', 'app/services')
-  add_group('Validators', 'app/validators')
-  add_group('Views', 'app/views')
-  enable_coverage(:branch)
-end
-
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(
-  [
-    SimpleCov::Formatter::HTMLFormatter,
-    SimpleCov::Formatter::Console
-  ]
-)
+# SimpleCov.start do
```

Comment:
No more simplecov is a good thing in my book but I guess it's unintentional here? :D

Reply:
Haha - yeah I comment it out locally cos it means I can't see the actual test results!
This was proabbly also a result of my inadvertant commit all, but in this case I thought I'd pushed the fix before requesting review.  Apparently not!